### DISCUSSION TITLE
How to implement the card flip animation using tailwind css?

### DISCUSSION THREAD

**Discussion by cupid20103** [tailwindlabs/tailwindcss#14084] [Help]
Hello, everyone!
Now I am trying to implement the card flip animation.
But the animation is not working. I used tailwind css like this but I am not sure this is works well.

`transition-all duration-500 [transform-style:preserve-3d] group-hover:[transform:rotateY(180deg)]`

This style works well on the inline css?

**Comment by wongjn** [comment]
Seems to be working [in this Tailwind Play](https://play.tailwindcss.com/UBtfBhk8d7), which would suggest it could be some misconfiguration or conflict within your project.
